db/migrate/20150718155400_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.text :content
+      t.integer :link_id
+
+      t.timestamps null: false
+    end
+  end
```

Comment:
We need to index comments by foreign keys :
add_index :comments, :link_id
add_index :comments, :user_id